### PR TITLE
Move rubygem-runcible from katello-packaging

### PR DESCRIPTION
**Pull request by ehelms**
For plugin updates, please indicate which repos this should be built into:

* [x] Nightly
* [ ] 1.16
* [ ] 1.15
* [ ] 1.14

See Foreman's [plugin maintainer documentation](http://projects.theforeman.org/projects/foreman/wiki/How_to_Create_a_Plugin#Release-strategies) for more information.

---

### CODE REVIEW

**Comment by ehelms** [comment]
Removal PR -- https://github.com/Katello/katello-packaging/pull/574

File: katello/rubygem-runcible/rubygem-runcible.spec [hunk 1]
```diff
@@ -0,0 +1,1272 @@
...
+%global gem_name runcible
+
+Name:           %{?scl_prefix}rubygem-%{gem_name}
+Version:        2.3.0
+Release:        1%{?dist}
+Summary:        A gem exposing Pulp's juiciest parts
+Group:          Applications/System
+License:        MIT
+URL:            https://github.com/Katello/runcible
+Source0:        http://rubygems.org/downloads/%{gem_name}-%{version}.gem
```
Comment (ekohl) [inline katello/rubygem-runcible/rubygem-runcible.spec]: https

File: katello/rubygem-runcible/rubygem-runcible.spec [hunk 2]
```diff
@@ -0,0 +1,1272 @@
...
+cp -a .%{gem_dir}/* \
+        %{buildroot}%{gem_dir}/
+mkdir -p %{buildroot}%{gem_docdir}
+
+%files
+%dir %{gem_instdir}
+%{gem_instdir}/lib
+%exclude %{gem_cache}
+%{gem_spec}
+%doc %{gem_instdir}/LICENSE
```
Comment (ekohl) [inline katello/rubygem-runcible/rubygem-runcible.spec]: %license

**Comment by ehelms** [comment]
Updated

File: katello/rubygem-runcible/rubygem-runcible.spec [hunk 3]
```diff
@@ -0,0 +1,1272 @@
...
+%files
+%dir %{gem_instdir}
+%{gem_instdir}/lib
+%exclude %{gem_cache}
+%{gem_spec}
+%license %{gem_instdir}/LICENSE
+
+%files doc
+%doc %{gem_docdir}
+%{gem_instdir}/Rakefile
```
Comment (ekohl) [inline katello/rubygem-runcible/rubygem-runcible.spec]: I think this and ```Gemfile``` can be left out entirely

**Comment by ehelms** [comment]
Updated

**Pull request merged**